src/Nox.Types/Types/TimeZoneCode/TimeZoneCode.cs

```diff
@@ -1,10 +1,227 @@
+using System.Collections.Generic;
+using System.Linq;
+
 namespace Nox.Types;
 
     /// <summary>
     /// Represents a Nox <see cref="TimeZoneCode"/> type and value object.
     /// </summary>
     /// <remarks>Placeholder, needs to be implemented</remarks>
-    // TODO Keep the List of Codes, find the proper ISO
-    public sealed class TimeZoneCode : ValueObject<string, TimeZoneCode>
+public sealed class TimeZoneCode : ValueObject<string, TimeZoneCode>
+{
+    private static readonly HashSet<string> _timeZoneCodes = new() { 
```

Comment:
Can you move this to an internal class, in the same folder, so keep this type smaller?